boards/gd32307c_eval.json

```diff
@@ -0,0 +1,53 @@
+{
+  "build": {
+    "core": "gd32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DGD32F307xC -DGD32F3 -DGD32F30x -DGD32F30X_CL",
+    "f_cpu": "120000000L",
+    "hwids": [
+      [
+        "0x1EAF",
+        "0x0003"
+      ],
+      [
+        "0x1EAF",
+        "0x0004"
+      ]
+    ],
+    "mcu": "gd32f307vct6",
+    "product_line": "GD32F307xG",
+    "variant": "GD32F307VG_MBED",
```

Comment:
Wait, if this is the board for the GD32F307VG_MBED variant then, then we need to delete https://github.com/CommunityGD32Cores/platform-gd32/blob/main/boards/gd32f307_mbed.json?  But the board file name is GD32307**C**, so is it really that VG board?

Reply:
The GD32F307V-MBED is a different board from the GD32307C-EVAL board.  Note our interesting naming convention for these boards- sometimes it is GD32[EF]xxx while most of the time it is GD32xxx.  No rhyme or reason that I can tell!  We currently have 72 demo boards - I have attached a document listing them all for your reference.  I only have a few of these so I'm starting with those.
[GigadeviceBoardList_tabbed.txt](https://github.com/CommunityGD32Cores/platform-gd32/files/6903156/GigadeviceBoardList_tabbed.txt)

Reply:
resolved via 9f8b7d8a7f0a8fcb2d31054041e0c36cf05547f1